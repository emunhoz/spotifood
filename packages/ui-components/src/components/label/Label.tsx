import React from 'react'
import * as S from './Label.style'

interface ILabel {
  value?: string
  htmlFor?: string
}

export const Label = ({ value, ...rest }: ILabel) => (
  <S.Label {...rest}>{value}</S.Label>
)
