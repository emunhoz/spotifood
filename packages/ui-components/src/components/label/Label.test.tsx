import React from 'react'
import { render } from '../../utils/test-utils'
import { Label } from './Label'

describe('[Component]: Label', () => {
  test('render the component', () => {
    const { asFragment } = render(<Label />)
    expect(asFragment()).toMatchSnapshot()
  })
})
