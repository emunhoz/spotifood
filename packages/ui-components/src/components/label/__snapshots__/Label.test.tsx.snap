// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: Label render the component 1`] = `
<DocumentFragment>
  <label
    class="sc-bdfBwQ fWEfIp"
  />
</DocumentFragment>
`;
